src/errors/error-view.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+
+class ErrorView extends Component {
```

Comment:
This is prolly something we could do elsewhere too, but when theres no `state` involved and only `props`, you can use a [stateless functional component](https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#stateless-functional-components). Less code ftw!